From c9c6a525c029fcaa45839c3b3039566f3e292f8d Mon Sep 17 00:00:00 2001
From: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@linux.intel.com>
Date: Mon, 21 May 2018 15:16:38 -0700
Subject: [PATCH 1494/2367] usb: Add info about USB_ROLE_SWITCH in Kconfig.

Add some help about USB_ROLE_SWITCH in Kconfig.

Tracked-On: PKT-679
Signed-off-by: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@linux.intel.com>
---
 drivers/usb/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/Kconfig b/drivers/usb/Kconfig
index cde4127..4e4b9bd 100644
--- a/drivers/usb/Kconfig
+++ b/drivers/usb/Kconfig
@@ -207,6 +207,6 @@ config USB_ULPI_BUS
 	  be called ulpi.
 
 config USB_ROLE_SWITCH
-	tristate
+	tristate "USB role switch support"
 
 endif # USB_SUPPORT
-- 
2.7.4

